src/witan/httpapi/components/activities.clj

```diff
@@ -160,17 +160,30 @@
                           {:partition-key file-id})
     (return-receipt id)))
 
+(defn- conform-metadata-updates
+  "This fn attempts to heal any differences between specs across the applications"
+  [m]
+  (log/debug "Conforming metadata updates:" m)
+  (let [m' (reduce-kv (fn [a k v]
+                        (assoc a k (s/unform k (s/conform k v)))) {} m)]
```

Comment:
what is `unform` doing here? And worthwhile calling k spec?

Reply:
`conform` then `unform` is a cheeky coercion.